pkg/agent/manager/storecache/cache_test.go

```diff
@@ -351,8 +350,7 @@ func TestUpdateEntries(t *testing.T) {
 				},
 			},
 			setUpdate: func(update cache.UpdateEntries) *cache.UpdateEntries {
-				e := update.RegistrationEntries["foh"]
-				updatedEntry := proto.Clone(e).(*common.RegistrationEntry)
+				updatedEntry := createTestEntry()
```

Comment:
Why do this instead of changing the require.Equal below to spiretest.AssertProtoListEqual?

Reply:
equals below uses [Record ](https://github.com/MarcosDY/spire/blob/66cd401abae46bd9b4c7718c527344e88add820a/pkg/agent/manager/storecache/cache.go#L17) that are not an struct 
same happens with [staleentries](https://github.com/MarcosDY/spire/blob/66cd401abae46bd9b4c7718c527344e88add820a/pkg/agent/manager/cache/cache.go#L124)
And ` spiretest.AssertProtoListEqual` requires a proto.Message, 
the ideal solution for this is refactor our code to not longer cache proto messages in favor to structs, 
and we'll do that in a separate PR once changes on LRU are merged